src/iuse_actor.cpp

```diff
@@ -701,6 +716,11 @@ void consume_drug_iuse::load( const JsonObject &obj )
             effects.push_back( load_effect_data( e ) );
         }
     }
+    if( obj.has_array( "damage_over_time" ) ) {
+        for( const JsonObject e : obj.get_array( "damage_over_time" ) ) {
```

Comment:
this should actually be readable directly as an array, once load_DoT_data is a member function instead of a static function.